tests/Unit/RouteMissingTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Tests\Unit;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Http\Request;
+use Livewire\Component;
+use Illuminate\Support\Facades\Route;
+
+class RouteMissingTest extends TestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        if (PHP_VERSION_ID < 70400) {
+            $this->markTestSkipped('Only applies to PHP 7.4 and above.');
+        }
+
+        if (! method_exists(\Illuminate\Routing\Route::class, 'missing')) {
+            $this->markTestSkipped('Need Laravel >= 8');
+        }
+    }
+
+    /** @test */
+    public function route_supports_laravels_missing_fallback_function(): void
+    {
+        $class = <<<'PHP'
+namespace Tests\Unit;
+use Livewire\Component;
+class ComponentWithModel extends Component
+{
+    public FrameworkModel $framework;
+}
+PHP;
+        eval($class);
```

Comment:
Why not just add the class below?

Reply:
because the test will fail with PHP 7.3 and below

we test multiple PHP version with a [matrix](https://github.com/livewire/livewire/blob/master/.github/workflows/test.yml#L16)